source/Tectia/index.rst

```diff
@@ -0,0 +1,589 @@
+.. _Tectia:
+
+######
+Tectia
+######
+
+RDHPCS users with a CAC who are logging in from a Windows, Mac, or Linux workstation/laptop are required to use CAC login. Access to RDHPCS resources via CAC requires a CAC reader and necessary software. The Tectia SSH Client software has been selected to meet the remote CAC login requirements for the RDHPCS program. Two licenses have been purchased for each RDHPCS user.
+
+The following features are supported:
+
+* Port forwarding
+* X11 tunneling
+
+Access to RDHPCS Systems from a system which cannot directly access a user's CAC is not supported.
+
+Tectia Initial Setup procedure
+------------------------------
+Host names for the CAC Bastion Server in Boulder, CO:
+
+.. code:: shell
+
+   bastion-jet.boulder.rdhpcs.noaa.gov
+   bastion-hera.boulder.rdhpcs.noaa.gov
+   bastion-niagara.boulder.rdhpcs.noaa.gov
+   bastion-gaea.boulder.rdhpcs.noaa.gov
+
+Host names for the CAC Bastion Server in Princeton, NJ:
+
+.. code:: shell
+
+   bastion-jet.princeton.rdhpcs.noaa.gov
+   bastion-hera.princeton.rdhpcs.noaa.gov
+   bastion-niagara.princeton.rdhpcs.noaa.gov
+   bastion-gaea.princeton.rdhpcs.noaa.gov
+
+
+The following OS-specific sections (Windows, Linux, MAC) describe how to do the following:
+
+* Download the Tectia software
+* Install the Tectia software on your local laptop or workstation
+* Install the license file on your local laptop or workstation
+* Configure the Tectia software
+* Use the client software to connect to R&amp;D HPC Systems
+* Set up port tunneling
+
+Configuring Tectia:  Windows
+----------------------------
+
+SSH Tectia Client provides secure terminal client functionality for remote users and system administrators for accessing remote hosts running SSH Tectia Server or other Secure Shell server. 
+
+These are the steps to install and configure Tectia for a Windows system:
+
+* Download and install the Tectia software on your local laptop or workstation
+* Install the license file on your local laptop or workstation. 
+
+   .. note::
+
+      Users will likely need to contact their local IT Help Desk to download and install Tectia.
+
+* Configure the Tectia software
+* Use the client software to connect to RDHPCS Systems
+* Set tunneling options
+
+**Requirements**
+
+* Your CAC information must be in AIM for the Tectia CAC logon to work. To verify AIM has your CAC information start a AIM help ticket requesting verification that your CAC information in in AIM. Account Management (AIM) Help - email - rdhpcs.aim.help@noaa.gov rdhpcs.aim.help@noaa.gov.
```

Comment:
Your CAC information... to work. To confirm your CAC information is correct in AIM, visit https://aim.rdhpcs.noaa.gov USING YOUR CAC TO AUTHENTICATE.   If your CAC information has been refreshed, wait 15 minutes before attempting to log in using Tectia.